.github/workflows/test_security-scan-workflow.yml

```diff
@@ -30,6 +30,10 @@ on:
 jobs:
   Default_Test-PlatSec-Workflow:
     uses: ./.github/workflows/platsec-security-scan-reusable-workflow.yml
+    # secrets:
+    #   RH_CATALOG_USERNAME: ${{ secrets.RH_CATALOG_USERNAME }}
+    #   RH_CATALOG_PASSWORD: ${{ secrets.RH_CATALOG_PASSWORD }}
+
```

Comment:
Tiny Nitpick (Feel free to ignore!):
Was this section being kept around for testing? Otherwise I would just remove it.